pkgs/by-name/gn/gnome-font-viewer/package.nix

```diff
@@ -27,6 +27,9 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  # Suppress incompatible function pointer error when building with clang.
```

Comment:
Thanks. If it is [this one](https://gitlab.gnome.org/GNOME/gnome-font-viewer/-/commit/565d795731471c27542bb9ee60820a2d0d15534e), can you add a comment that it is fixed in 47.alpha so that we do not forget to remove it?

Reply:
Would it be better to cherry-pick the commit instead?

Reply:
I switched it to use the patch from upstream.